site/public/css/global.css

```diff
@@ -80,10 +80,6 @@ nav a > i.f {
     padding-left: 7px;
 }
 
-.breadcrumb_heading {
-    font-size: 24px;
-}
-
```

Comment:
The removal of this makes breadcrumbs look strange:
![image](https://user-images.githubusercontent.com/28243927/134845708-38521c05-1df8-4604-8ccd-60ad61d6de88.png)

Reply:
Good catch.  I used PHPStorm to look for each of these and apparently the indexing doesn't work as well as I'd hope.  Corrections made.